panther_gazebo/launch/spawn.launch.py

```diff
@@ -0,0 +1,298 @@
+            executable="parameter_bridge",
+            name="gz_bridge",
+            parameters=[{"config_file": gz_bridge_config_path}],
+            namespace=robot_name,
+            output="screen",
+        )
+
+        bringup_launch = IncludeLaunchDescription(
+            PythonLaunchDescriptionSource(
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("panther_bringup"),
+                        "launch",
+                        "bringup.launch.py",
+                    ]
+                )
+            ),
+            launch_arguments={
+                "wheel_type": wheel_type,
+                "wheel_config_path": wheel_config_path,
+                "controller_config_path": controller_config_path,
+                "battery_config_path": battery_config_path,
+                "publish_robot_state": publish_robot_state,
+                "use_sim": "True",
+                "simulation_engine": "ignition-gazebo",
+                "namespace": robot_name,
+            }.items(),
+        )
+
+        ns_prefix = robot_name + "/" if robot_name else robot_name
+
+        map_transform = Node(
+            package="tf2_ros",
+            executable="static_transform_publisher",
+            name="static_tf_publisher",
+            namespace=robot_name,
+            output="screen",
+            arguments=[x, y, z, roll, pitch, yaw, "map", ns_prefix + "odom"],
+            condition=IfCondition(add_map_transform),
+        )
+
+        # bringup.launch.py has a timerAction in it. If the timerAction in simulation.launch.py ​​is smaller than bringup.launhc.py, the namespace will be overwritten, resulting creating nodes with the same namespace.
+        group = TimerAction(
+            period=10.0 * idx,
+            actions=[
+                spawn_log,
+                spawn_robot,
+                gz_bridge,
+                bringup_launch,
+                map_transform,
+            ],
+        )
+        spawn_group.append(group)
+
+    return spawn_group
+
+
+def generate_launch_description():
+    wheel_type = LaunchConfiguration("wheel_type")
+    declare_wheel_type_arg = DeclareLaunchArgument(
+        "wheel_type",
+        default_value="WH01",
+        description=(
+            "Specify the type of wheel. If you select a value from the provided options ('WH01',"
+            " 'WH02', 'WH04'), you can disregard the 'wheel_config_path' and"
+            " 'controller_config_path' parameters. If you have custom wheels, set this parameter"
+            " to 'CUSTOM' and provide the necessary configurations."
+        ),
+        choices=["WH01", "WH02", "WH04", "CUSTOM"],
+    )
+
+    declare_wheel_config_path_arg = DeclareLaunchArgument(
+        "wheel_config_path",
+        default_value=PathJoinSubstitution(
+            [
+                FindPackageShare("panther_description"),
+                "config",
+                PythonExpression(["'", wheel_type, ".yaml'"]),
```

Comment:
```suggestion
                PythonExpression([wheel_type, ".yaml"]),
```
Why the single quotes are here?

Reply:
Does not work